decidim-admin/app/views/decidim/admin/attachments/index.html.erb

```diff
@@ -1,37 +1,47 @@
-<section id="attachments">
-  <h4><%= t(".attachments_title", scope: "decidim.admin") %></h4>
+<div class='card' id="attachments">
+  <div class="card-divider">
+    <h2 class="card-title">
+      <%= t(".attachments_title", scope: "decidim.admin") %>
+      <% if can? :create, authorization_object %>
+        <%= link_to t("actions.new", scope: "decidim.admin", name: t("models.attachment.name", scope: "decidim.admin")), url_for(action: :new), class: 'button tiny button--title new' %>
+      <% end %>
+    </h2>
+  </div>
 
-  <% if can? :create, authorization_object %>
-    <div class="actions title">
-      <%= link_to t("actions.new", scope: "decidim.admin", name: t("models.attachment.name", scope: "decidim.admin")), url_for(action: :new), class: 'new' %>
-    </div>
-  <% end %>
+  <div class="card-section">
+    <% if attached_to.attachments.any? %>
+      <div class="table-scroll">
+        <table class="table-list">
+          <thead>
+            <tr>
+              <th><%= t("models.attachment.fields.title", scope: "decidim.admin") %></th>
+              <th><%= t("models.attachment.fields.content_type", scope: "decidim.admin") %></th>
+              <th class="actions"></th>
+            </tr>
+          </thead>
+          <tbody>
+            <% attached_to.attachments.each do |attachment| %>
+              <tr data-id="<%= attachment.id %>">
+                <td>
+                  <%= link_to translated_attribute(attachment.title), polymorphic_path([attached_to, attachment]) %><br />
```

Comment:
As we've discussed offline we will prioritize icon links for actions outside the admin panel and the title links otherwise.